### ISSUE TITLE
Merge in legacy projects

### ISSUE THREAD

**Issue by dsernst** [Boundless-Avocado/Boundless-Avocado#67]
AWESOME work by the other teams!!! Let's merge some stuff in!
- [ ] 1. Ionic app  
- [ ] 2. Signup over text  
- [ ] 3. Invite over text  
- [ ] 4. Sending messages to group  
- [ ] 5. Reply to SMS messages to send back to group  
- [ ] 6. Nearby groups
- [ ] 7. Leave groups  
- [ ] 8. Password protected groups

What else am I missing?

INCREDIBLE JOB! @Boundless-Avocado/wicked-babies @Boundless-Avocado/aberrant-marble

**Comment by rmi22186** [comment]
Marble has taken care of 1, 4, 5, 7, and pretty sure we're closing in on 6. Babies has 2, 3, 8, and if I recall correctly, versions of 4 as well.

Would love to add muting notifications for individual rooms, people, or mute the app completely similar to iOS DND feature. Maybe even the ability to set how long the notifications are muted for.

**Comment by eds101** [comment]
I fixed the back button issue on our frontend, and I got nearbyGroups
working (although we should arguably remove that feature for the
foreseeable future anyway). Hope to merge in Facebook login
tomorrow.  Personally,  I'm interested in just requiring Fbook login so we
can get real, validated names. Then we'd know who we're talking to and not
have to deal w bad spam. I also gotta get gulpfile set up for prod w concat
&minification, and fix up msg formats (need to see group names).

Agree muting of a group or a user or the whole app for an hour or a day or
indefinitely would be great to have too.

Also i do want to set up private groups w admin(s).

**Comment by kchia** [comment]
Yeah, Babies has 2,3, and 8 (and technically 7 as well). I don't believe
our backend work is saved within the Boundless-Avocado/Boundless-Baby repo
yet. Will look into this and get back to everyone.

Thanks all,
Hou

**Comment by dsernst** [comment]
Ok here's an updated list:

Marble's written code for:
- [ ] 1. Ionic app
- [ ] 4. Sending messages to group
- [ ] 5. Reply to SMS messages to send back to group
- [ ] 6. Nearby groups
- [ ] 7. Leave groups

Babies's written code for:
- [ ] 2. Signup over text
- [ ] 3. Invite over text
- [ ] 8. Password protected groups

**Comment by dsernst** [comment]
Also btw everyone @eds101 and I were able to secure the domain name [guac.io](guac.io) ! :tada:
